hicommonwealth/commonwealth
# Activity Updates

## Link to Issue
Closes: #8438

## Description of Changes
- Adds `activity_rank_date` used to rank threads on activity dashboards
- Optimized activity query by creating a CTE that enables Postgres to fetch thread metadata and thread comments in parallel
- Migrated feed requests to React Query (removes doubled requests)
- Removed a lot of legacy UserDashboardRow components that are no longer necessary now that the Chain feed is gone.
- Created configurable default load test scenarios
- Updated `k6` bash script to use flags
- Exposed the load-testing utility routes


## Test Plan
- View for you and global activity tabs
- Create a thread
- View for you tab (new thread should not show up)
- Sign in on another account in another tab/browser
- Create a thread in a community that your original user has joined
- View the for you and global tab with the original user (the thread should only show up in the global tab)

## Deployment Plan

1. 

## Other Considerations

-
